packages/user-group-service/src/users/typedef.graphql

```diff
@@ -108,16 +157,21 @@ type Mutation {
   addUserFromLDAP(uid: String!): UserType
 }
 
+
 type Query {
   # Fetches specific User by uid, rhatUUID, apiRole, name
   getUsersBy(
     uid: String
     rhatUUID: String
     apiRole: String
     name: String
-  ): [UserType]
+  ): [UserType] @deprecated(reason: "This query is no longer maintained")
   # Fetches all Users
-  listUsers: [UserType]
+  listUsers: [UserType] @deprecated(reason: "This query is no longer maintained")
   # Fetch LDAP Group Members
-  getGroupMembers(cn: String): [LdapType]
+  getGroupMembers(cn: String): [LdapType] @deprecated(reason: "This query is no longer maintained")
+  # Fetches specific User by uid, rhatUuid
+  user(uid: String, rhatUuid: String): RoverUserType
```

Comment:
```suggestion
  user(uid: String, rhatUUID: String): RoverUserType
```

Inconsistent field Name with other queries

Reply:
The Rover field in the upstream (i.e. rover api) is called `rhUuid` so if anything, the field we had before is inconsistent. With the newer changes, I think it's a good idea to align the field names to avoid any unnecessary confusion.